Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,611 @@
+ The developer of such application/product will be responsible for
+ performing the validation processes described in FDA published
+ guidelines for the development of software-related medical devices."
+```
+
+For mode details see the page [FDA Guidelines for Software Developement](https://itk.org/Wiki/FDA_Guidelines_for_Software_Development).
+
+## Language bindings
+
+### What does `Error opening hint file NOTFOUND` mean?
+
+When you enable `USE_VTK` in the CMake configuration of ITK, and VTK
+was built with wrapping, CMake looks for a "hints" file.
+
+The "hints" file on the VTK source tree is found at: https://gitlab.kitware.com/vtk/vtk/-/blob/master/Common/Core/vtkWrappingHints.h
+
+Please switch CMake to `Advanced`, look for the `VTK_WRAP_HINTS`
+variable and set it to the full path to the "hints" file including the
+`"hints" filename itself.
+
+Here is how it looks in the `CMakeCache.txt` of a typical ITK
+installation:
+
+```
+//Where can the hints file be found
+VTK_WRAP_HINTS:FILEPATH=/home/johndoe/src/VTK/Wrapping/hints
+```
+
+### How do I use ITK with WrapITK?
+
+WrapITK allows you to use ITK from other languages such as Python, Java, or
+Ruby). See the page about [ITK wrappings](https://itk.org/Wiki/ITK/Wrapping).
+
+### How do I use ITK from C# .NET?
+
+There are currently three options:
+
+1. Use [ManagedITK](https://code.google.com/archive/p/manageditk/)
+   - **NOTE: This option is not recommended as the project is no
+     longer supported**.
+1. Use the [SimpleITK](https://simpleitk.org/) C# language
+   bindings:
+   - SimpleITK is great, but it does not support all filters.
+   - A list of available filters can be found
+     [here](http://www.itk.org/SimpleITKDoxygen/html/Filter_Coverage.html)
+     (valid Sept 2012).
+1. Manually wrap the desired functionality
+   - This option requires the most work, but is the most flexible,
+     allowing you to use any ITK functionality you desire
+   - An example can be found [here](https://itk.org/Wiki/ITK/Using_ITK_from_.NET).
+
+NOTE: `WrapITK` does not currently support C# language bindings, but could
+be made to do so.
+
+## Using the Insight Toolkit
+
+### How do I combine ITK and VTK in my application?
+
+A full tutorial on how to combine ITK and VTK is available at [Getting Started with ITK and VTK](http://www.itk.org/CourseWare/Training/GettingStarted-II.pdf).
+
+You will need the auxiliary classes available at the [ITKApps](https://github.com/InsightSoftwareConsortium/ITKApps)
+repository.
+
+For example, for converting ITK images into VTK images and back, you
+will find useful the files from the [ITKVTKGlue](https://github.com/InsightSoftwareConsortium/ITKVtkGlue)
+module:
+
+```
+itkImageToVTKImageFilter.h
+itkImageToVTKImageFilter.hxx
+itkVTKImageToImageFilter.h
+itkVTKImageToImageFilter.hxx
+```
+
+### How do I configure Emacs for ITK coding style?
+
+You can add the following codes into your `.emacs` file:
```

Comment:
Not sure about this section. I do not use emacs.

Reply:
I think it is likely no longer correct since our clang-format style update.

Reply:
Removed in 894e5e6.